configuration/xrock_gui/MultiDBConfig.yml

```diff
@@ -0,0 +1,10 @@
+main_server:
+  type: Serverless
+  path: modkom/component_db
+import_servers:
+  - type: Serverless
+    path: database/atomic_component_db
+    graph: graph_test
+  - type: Client
+    url: http://localhost:8183
+    graph: graph_production
```

Comment:
Is this the same as in xdbi?
I suggest that we use this opportunity to make the keys a bit clearer. How about:
main_server -> read_write_dbi
import_server -> read_only_dbi